ansible/idr-proxy.yml

```diff
@@ -37,6 +37,18 @@
         }}:8000
     when: "{{ groups[idr_environment | default('idr') + '-a-dockermanager-hosts'] is defined }}"
 
+  - name: Get mineotaur IP
+    set_fact:
+      # Optional, so that it can run in an environment where there's only
+      # an OMERO server but no analysis platform
+      omero_mineotaur_host_ansible: >-
+        {{
+          hostvars[groups[
+            idr_environment | default('idr') + '-a-dockermanager-hosts'][0]]
```

Comment:
Depending on your plans for deploying mineotaur you might want to add `(-a)?-mineotaur` as a hostgroup, though this is fine for now.